docker-compose.yaml

```diff
@@ -0,0 +1,33 @@
+version: '2'
+
+services:
+  postgres:
+      image: postgres:alpine
+      restart: always
+      ports:
+        - "5432:5432"
+  php:
+    build: docker/php
+    tty: true
+    labels:
+      kompose.service.type: nodeport
+    ports:
+      - 9000:9000
+    volumes:
+      - ./app:/app
+  nginx:
+      image: nginx:alpine
+      ports:
+        - "80:80"
+      volumes:
+         - ./app:/app
+         - ./docker/nginx/site.conf:/etc/nginx/conf.d/default.conf
+  15-puzzle-go:
+      build: 15-puzzle-go
+      command: go get -u github.com/nsf/termbox-go .
```

Comment:
Shouldn't it be in the Dockerfile?

Reply:
I've made a PR in the 15-puzzle-go to fix that: https://github.com/marmelab/15-puzzle-go/pull/13